ts_develop/conda/meta.yaml

```diff
@@ -26,7 +26,7 @@ requirements:
     - lsst-documenteer-pipelines =0.7
     - lxml =4.9
     - moto =3
-    - mypy =1.0.0
+    - mypy =1.2.1
```

Comment:
Any particular reason for this to be 1.2.1 and the other 1.2.0? shouldn't they be the same?

Reply:
Thanks for catching this typo. It needs to be `1.2.0` as well.